### ISSUE TITLE
Feature requests.

### ISSUE THREAD

**Issue by RiffyDiffy** [syrupyy/cookiecomiccreator#1]
Hi, there!

I'm one of the admins from the Cookie Run: Kingdom Wiki, and I've seen many people using your website to create comics in the wiki forums.

But, sadly, it has some limitations.

Here is a list of the many missing features:
- Rotation and x-y axis scaling tool.
- Jellies and treasures.
- Obstacles.
- Effects like gaussian blur, noise, glitch, etc... (Of course, you don't need to implement that, if that's okay for you.)

That's all I have to say, it would be even more cool if you added the following features for a more accessible way of creating comics.

Thanks for reading this!
-RiffyDiffy

**Comment by syrupyy** [comment]
hmmmm yes, this looks somewhat like my current to-do list, though mine also has adding NPCs and looking into ability effects since those were also requested. I don't know how possible blur and other effects would be in JavaScript but it's worth looking into, I've wanted to add things like that to my comics... at the end of the day, anything I don't add can always be made up for with image editing programs.
thank you for compiling suggestions, I'll use this!

**Comment by RiffyDiffy** [comment]
Thanks so much for adding the NPC's!

...Also, bonus note: I was the person who ripped Choco Werehound Brute, Dark Enchantress Cookie, Jelly Wormy, and the Ferryman's sprites.

**Comment by syrupyy** [comment]
aha yeah! I was about to ask you, how did you make those dark enchantress sprites? it said they were "assembled with paint.net", did you get the original spritesheet and put it back together by hand

**Comment by RiffyDiffy** [comment]
Yeah, that's what I did when starting out posting cutscene sprites, but now, I use Spine (The program used to make the animations in CRK) to rip the sprites.

**Comment by syrupyy** [comment]
sorry for asking so many questions but this could be useful to me, is there a program you used on top of Spine? I noticed you also posted voice lines to the wiki, those were previously thought to be encrypted-

**Comment by RiffyDiffy** [comment]
For the voice lines, I turn off the music and sound effects and then record the lines on my phone.
Then, I import the footage in Google Drive and I download it on my PC.
Next, I use Audacity to separate the lines that were recorded in the footage, save them as .OGG files, and upload the files in the wiki.

This is the same technique I use with Sonic Cookie's loading lines, GingerBrave's English lines, some Pre-gacha and gacha lines, and so on.

Yes, the raw files are still encrypted, but the only way to get them is to record it on a phone (iOS or Android, whatever you prefer.) and use a program to seperate them.

**Comment by syrupyy** [comment]
ahhhh!! I see, I see
I do hope methods for this can be better eventually, as it stands using code to rip files straight from Kingdom like this does with Ovenbreak rn would be nigh impossible
